packages/agoric-cli/lib/helpers.js

```diff
@@ -0,0 +1,39 @@
+export const makePspawn = ({
+  pspawnEnv = process.env,
+  log = console,
+  chalk,
+  spawn,
+}) => (cmd, cargs, { stdio = 'inherit', env = pspawnEnv, ...rest } = {}) => {
```

Comment:
Chip or Brian would likely be bothered by the partial application on this line. I did have a double-take to see it myself.

Reply:
I'll give that a function name.

Reply:
I wrote some types around this function, too.  Hope that helps with clarity.